### PR TITLE
feat: add workflow to update external docs

### PR DESCRIPTION
**Pull request by bjohansebas**
To avoid having to update the documentation like in #1555 

see: https://github.com/peter-evans/create-pull-request/tree/v5?tab=readme-ov-file#action-inputs

### CODE REVIEW

File: .github/workflows/update-external-docs.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    update-docs:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Run scripts
+              run: bash ./get-contributing.sh && bash ./get-readmes.sh
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v5
+              with: 
+                token: ${{ secrets.ACTIONS_BOT_TOKEN }}
```
Comment (UlisesGascon) [inline .github/workflows/update-external-docs.yml]:
```suggestion
                token: ${{ secrets.GITHUB_TOKEN }}
```

Is the automatic token an option in this case? https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Comment (bjohansebas) [reply .github/workflows/update-external-docs.yml]: Yes, it is the default option for the token, and now that we know it is created automatically, it is a good idea to leave it as you suggested.

**Review by UlisesGascon — APPROVED** [review]
LGTM!

**Comment by crandmck** [comment]
I'm relying on review from @UlisesGascon to ensure that this doesn't break anything and works as expected!

Thanks @bjohansebas

**Pull request merged**